src/guides/v2.3/frontend-dev-guide/templates/template-email-migration.md

```diff
@@ -0,0 +1,230 @@
+---
+group: frontend-developer-guide
+title: Migrating custom email templates
+functional_areas:
+  - Frontend
+---
+
+{% raw %}
+With the release of Magento 2.3.4, we made some changes to custom email templates and how they access data and methods for email content.
+This topic describes the changes and provides instructions on how to convert your existing custom email templates.
+
+## Changes to the custom email template workflow
+
+In Magento 2.3.4, custom email templates are only allowed to use scalar values for variable data.
+Direct calls to methods are no longer allowed.
+To be more specific, methods can no longer be called from variables from either the `var` directive or when used as parameters.
+For example `{{var order.getEmailCustomerNote()}}` or `{{something myVar=$obj.method()}}` will fail to resolve.
+
+A 'custom email template' is any new template created in the Magento admin **Marketing** > Communications > **Email Templates** > **Add New Template** area.
+Notice in the incorrect example, the `getConfirmationLink()` method is called directly.
+
+-  Old way: `{{var subscriber.getConfirmationLink()}}`
+-  New way: `{{var subscriber_data.confirmation_link}}`
+
+We refer to this as 'strict mode' for email templates.
+All default templates have been converted to this strict mode.
+
+All existing custom email templates will continue to work after upgrading to 2.3.4.
```

Comment:
I think this may be worthy of calling out more, maybe in one of the pull quote style info boxes?

Reply:
Good idea.